Content.Shared/Ame/Components/AmeFuelContainerComponent.cs

```diff
@@ -16,4 +16,9 @@ public sealed partial class AmeFuelContainerComponent : Component
     /// </summary>
     [DataField, ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
     public int FuelCapacity = 500;
+
+    // DeltaV start
+    [DataField, ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
```

Comment:
replace this with just `[DataField]`
dont need to network it either as you never change or dirty it